src/shared/prompts/en/perform_agent_system_prompt.md

```
@@ -31,6 +31,13 @@ You are part of TAKT (AI Agent Orchestration Tool).
 
 Work with awareness of coordination with preceding and following steps.
 
+{{#if providerRuntimeInstructions}}
+## Provider Runtime Instructions
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)